packages/office-ui-fabric-react/src/components/Panel/examples/Panel.Navigation.Example.tsx

```diff
@@ -1,8 +1,7 @@
 import * as React from 'react';
-import { DefaultButton, IconButton } from 'office-ui-fabric-react/lib/Button';
```

Comment:
I didn't see anything about an IconButton on the bugs list... which bug is this addressing?

Reply:
Bugs 2, 5, 8, and 11 were all targeting this icon button.

They are all legitimate issues, but the example is not materially weaker by removing this control altogether.